src/MemberList.h

```diff
@@ -0,0 +1,62 @@
+// SPDX-FileCopyrightText: 2021 Nheko Contributors
+//
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+#pragma once
+
+#include "CacheStructs.h"
+#include <QAbstractListModel>
+
+class MemberList : public QAbstractListModel
```

Comment:
It might be nice to connect this to any member state changes in the room and in the future to also show invites and knocks. Doesn't have to happen now though.

Reply:
Not planning to do this here or now.